docs/CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 ## CLI
 
 ## Catalog, Lambdas
+* [Changed] `top_hash`-based package routes (backwards compatible) ([#1938](https://github.com/quiltdata/quilt/pull/1938))
```

Comment:
This is reasonable behavior but I'm wondering if forwarding to a dedicated `/h/` or whatever hash route is more correct in terms of traceability? Please make a note that /timestamp URLs are not forwarded in any way, but that the user remains at /timestamp.

Reply:
To be clear I am totally cool with current behavior (honor both types of routes, no redirect). I am just asking if anything becomes cleaner or safer with two routes.